android/app/src/main/java/com/happy/friendogly/presentation/ui/playground/PlaygroundFragment.kt

```diff
@@ -0,0 +1,1015 @@
+        binding.rcvPlaygroundPetSummary.addItemDecoration(
+            object : RecyclerView.ItemDecoration() {
+                override fun getItemOffsets(
+                    outRect: Rect,
+                    view: View,
+                    parent: RecyclerView,
+                    state: RecyclerView.State,
+                ) {
+                    super.getItemOffsets(outRect, view, parent, state)
+                    val position = parent.getChildAdapterPosition(view)
+                    if (position != 0) {
+                        val overlapOffset =
+                            (10f * requireContext().resources.displayMetrics.density).toInt()
+                        outRect.left = overlapOffset * -1
+                    }
+                }
+            },
+        )
+    }
+
+    private fun setupActivityResultLauncher() {
+        activityResultLauncher =
+            registerForActivityResult(ActivityResultContracts.StartActivityForResult()) { result ->
+                if (result.resultCode == Activity.RESULT_OK) {
+                    val messageUpdated =
+                        result.data?.getBooleanExtra(EXTRA_MESSAGE_UPDATED, DEFAULT_MESSAGE_UPDATED)
+                            ?: return@registerForActivityResult
+
+                    if (messageUpdated) {
+                        val myPlayground =
+                            viewModel.myPlayground.value ?: return@registerForActivityResult
+                        viewModel.loadPlaygroundInfo(myPlayground.id)
+                    }
+                }
+            }
+    }
+
+    private fun setupBottomSheetBehavior() {
+        bottomSheetBehavior.isDraggable = true
+        bottomSheetBehavior.isHideable = false
+        bottomSheetBehavior.isFitToContents = false
+        bottomSheetBehavior.expandedOffset = 50
+        bottomSheetBehavior.peekHeight = 250
+
+        bottomSheetBehavior.addBottomSheetCallback(
+            object : BottomSheetBehavior.BottomSheetCallback() {
+                override fun onSlide(
+                    bottomSheet: View,
+                    slideOffset: Float,
+                ) {
+                    if (viewModel.uiState.value == PlaygroundUiState.FindingPlayground && slideOffset >= 0.4f) {
+                        viewModel.updateUiState(PlaygroundUiState.ViewingPlaygroundInfo)
+                    }
+                }
+
+                override fun onStateChanged(
+                    bottomSheet: View,
+                    newState: Int,
+                ) {
+                    if (viewModel.uiState.value == PlaygroundUiState.ViewingPlaygroundInfo &&
+                        newState == BottomSheetBehavior.STATE_COLLAPSED
+                    ) {
+                        changeRecentlyClickedMarkerSize()
+                        binding.rcvPlaygroundPet.smoothScrollToPosition(0)
+                    } else {
+                        viewModel.updateRefreshBtnVisibility(false)
```

Comment:
```suggestion
                        binding.btnWoofFootprintRefresh.isVisible = false
```

바로 activity에서 뷰를 바꿀 수 있네요..! 
`activity -> viewModel -> xml -> bindingAdapter` 로 불필요한 과정을 거치는 것 같습니다!